src/MobileRedirection.php

```diff
@@ -180,6 +186,41 @@ public function redirect() {
 		}
 	}
 
+	/**
+	 * Add mobile version switcher link/styles when mobile redirection is not enabled.
+	 */
+	public function maybe_add_mobile_switcher_hooks() {
+		if ( $this->is_mobile_request() ) {
+			$this->add_mobile_switcher_head_hooks();
+			$this->add_a2a_linking_hooks();
+			$this->add_mobile_switcher_footer_hooks();
```

Comment:
Please then also make use of these in the `redirect` method.

Reply:
Yes, it's on my todo!